app/src/main/java/com/example/evently/EntrantListAdapter.java

```diff
@@ -0,0 +1,62 @@
+package com.example.evently;
+
+import java.util.ArrayList;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.evently.data.model.Account;
+
+/**
+ * Recycler view that displays each entrant as a row with their profile picture and name
+ */
+public class EntrantListAdapter extends RecyclerView.Adapter<EntrantListAdapter.ViewHolder> {
+    private ArrayList<Account> entrants;
+    private LayoutInflater inflater;
+
+    EntrantListAdapter(Context context, ArrayList<Account> entrants) {
+        this.entrants = entrants;
+        this.inflater = LayoutInflater.from(context);
+    }
+
+    public static class ViewHolder extends RecyclerView.ViewHolder {
+        TextView image;
+        TextView name;
+
+        public ViewHolder(View view) {
+            super(view);
+            // Can define click listeners here
+
+            this.image = view.findViewById(R.id.image_placeholder);
```

Comment:
Is there a requirement for having images for entrant profiles btw? I didn't see such a thing.

Of course it'd be nice to have but it's likely not something we need to put in at the moment.